.github/ISSUE_TEMPLATE/todo-ospo-ambassador-request.md

```diff
@@ -9,14 +9,14 @@ assignees: anajsana
 # TODO OSPO Ambassador Request
 
 This is an issue created to request to become an OSPO Ambassador. If you (as individual) are interested to apply, please make sure you have (1) read the [TODO OSPO Ambassador Program documentation](https://github.com/todogroup/governance/blob/main/TODO-OSPO-Ambassador-Program.md)
-and (2) fulfil the requirements from the Pre-submission checklist.
+and (2) fulfill the requirements from the pre-submission checklist.
```

Comment:
Pre-submission checklist should remain capitalized as it is the name of a list (line 14) and not just a description.